src/apns.erl

```diff
@@ -118,7 +121,7 @@ connect(Name, Fun) when is_function(Fun, 2) ->
 connect(Name, ErrorFun, FeedbackFun) ->
   connect(
     Name, (default_connection())#apns_connection{error_fun    = ErrorFun,
-                                                 feedback_fun = FeedbackFun}).
+                                                       feedback_fun = FeedbackFun}).
```

Comment:
Line 124 is too long:                                                        feedback_fun = FeedbackFun})..